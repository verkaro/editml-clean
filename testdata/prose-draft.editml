The Mariner 9 mission was a {=landmark=} achievement in planetary science. {-It was the first spacecraft to orbit another planet, arriving at Mars in 1971.-}{+Arriving at Mars in 1971, it was the first spacecraft to orbit another planet.+}{>Check the exact date of orbit insertion.<}

{move~Its primary objective was to map 70% of the Martian surface.~obj}The probe vastly exceeded expectations. {move:obj} It also studied the planet's two moons, Phobos and Deimos, in {+unprecedented+} detail.
