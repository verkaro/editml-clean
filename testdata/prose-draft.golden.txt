The Mariner 9 mission was a landmark achievement in planetary science. Arriving at Mars in 1971, it was the first spacecraft to orbit another planet.

The probe vastly exceeded expectations. Its primary objective was to map 70% of the Martian surface. It also studied the planet's two moons, Phobos and Deimos, in unprecedented detail.
